src/unistd.rs

```diff
@@ -737,11 +737,23 @@ pub fn write(fd: RawFd, buf: &[u8]) -> Result<usize> {
     Errno::result(res).map(|r| r as usize)
 }
 
+/// Directive that tells lseek how to reposition the offset of the file.
 pub enum Whence {
+    /// The file offset shall be set to offset bytes.
     SeekSet,
+    /// The file offset shall be set to its current location plus offset.
     SeekCur,
+    /// The file offset shall be set to the size of the file plus offset.
     SeekEnd,
+    /// Adjust the file offset to the next location in the file greater than
```

Comment:
Data is supposed to mean a part of the file that's been written to. Maybe it should be called 'some data'?